### PR TITLE
[MultiFun] Prepare for new hardware

### PR DESCRIPTION
**Pull request by szczukot**

### CODE REVIEW

**Comment by szczukot** [comment]
pls merge

**Comment by gizmocuz** [comment]
What is multifun ? As there seem to be so many changes, i guess this is meant for your private git repo and not here ?
Is there a forum thread about this ?

**Comment by szczukot** [comment]
I always add link to product in header (// implememtation for regulators : https://www.recalart.com/)
I don't have this device, but some users asked me directly about this for them.

**Comment by gizmocuz** [comment]
Is there a forum thread about this ?
I also see a lot of Polish texts being used in the class... will this only be used in poland ?

**Comment by szczukot** [comment]
I didn't create post on forum. if it is necessary I will ask someone to add it.
I have only polish documentation about this device, and i copy-paste messages from it. I will ask the producer of the English version or I translate it to English.

**Comment by gizmocuz** [comment]
Thanks

File: msbuild/domoticz.vcxproj.filters [hunk 1]
```diff
@@ -438,6 +438,8 @@
     <Filter Include="Devices\HEOS">
       <UniqueIdentifier>{43faa747-76f8-47a8-a90b-ebcb30a1f72e}</UniqueIdentifier>
     </Filter>
+    <Filter Include="Devices\MultiFun">
+      <UniqueIdentifier>{61f9e7d0-4204-474d-b953-b8d55b281296}</UniqueIdentifier>
```
Comment (gizmocuz) [inline msbuild/domoticz.vcxproj.filters:442]: Did you open the visual studio project ... in visual studio ?
It was broken.... for example you missed the closing tag (</Filter)
Also below for include/compile

File: msbuild/domoticz.vcxproj.filters [hunk 2]
```diff
@@ -1903,6 +1905,8 @@
     <ClInclude Include="..\hardware\HEOS.h">
       <Filter>Devices\HEOS</Filter>
     </ClInclude>
+    <ClInclude Include="..\hardware\MultiFun.h">
+      <Filter>Devices\MultiFun</Filter>
```
Comment (gizmocuz) [inline msbuild/domoticz.vcxproj.filters:1909]: Missing `</ClInclude>`

File: msbuild/domoticz.vcxproj.filters [hunk 3]
```diff
@@ -2499,6 +2503,8 @@
     <ClCompile Include="..\hardware\HEOS.cpp">
       <Filter>Devices\HEOS</Filter>
     </ClCompile>
+    <ClCompile Include="..\hardware\MultiFun.cpp">
+      <Filter>Devices\MultiFun</Filter>
```
Comment (gizmocuz) [inline msbuild/domoticz.vcxproj.filters:2507]: Missing `</ClCompile>`

**Review by gizmocuz — COMMENTED** [review]
Missing tags caused VStudio project to be broken

**Comment by szczukot** [comment]
thanks
had to be a conflict with Heos

**Pull request merged**